roadmap/implementors-guide/subsystems-overview.md

```diff
@@ -0,0 +1,95 @@
+# Subsystems Overview
+
+These subsystems comprise the Validity module. All detailed documentation lives within the guide; this document simply attempts to enumerate and briefly describe each subsystem and the map of their communications.
```

Comment:
Can you elaborate on where you want to draw the line between the Validity modules and other modules? Would it be inappropriate to add something like Bitfield Signing, Availability Distribution, or Bitfield Distribution in this document? Those are some examples of subsystems that are necessary based on the expectations of the runtime.

Reply:
Let me make sure that I understand what we're talking about. Bitfields govern whether a given validator is currently available, or assigned to a parachain or parathread, right? If that's correct, then I think that everything having to do with bitfields should not be in the Validity module. Possibly there should exist a Para Scheduling module?

I want to keep this module tightly focused on the following problem: given unbounded parablock candidates being gossiped around, how can we choose one that a quorum of validators agrees is valid, while minimizing total validation work performed?